tests/Unit/Evolution/Systems/ScalarWave/Test_Equations.cpp

```diff
@@ -50,48 +65,75 @@ void check_du_dt(const size_t npts, const double time) {
     return coords;
   }();
 
-  auto box = db::create<db::AddSimpleTags<
-      Tags::dt<ScalarWave::Pi>, Tags::dt<ScalarWave::Phi<Dim>>,
-      Tags::dt<ScalarWave::Psi>, ScalarWave::Pi,
-      Tags::deriv<ScalarWave::Pi, tmpl::size_t<Dim>, Frame::Inertial>,
-      Tags::deriv<ScalarWave::Phi<Dim>, tmpl::size_t<Dim>, Frame::Inertial>>>(
-      Scalar<DataVector>(pow<Dim>(npts), 0.0),
-      tnsr::i<DataVector, Dim, Frame::Inertial>(pow<Dim>(npts), 0.0),
-      Scalar<DataVector>(pow<Dim>(npts), 0.0),
-      Scalar<DataVector>(-1.0 * solution.dpsi_dt(x, time).get()),
-      [&x, &time, &solution ]() noexcept {
-        auto dpi_dx = solution.d2psi_dtdx(x, time);
-        for (size_t i = 0; i < Dim; ++i) {
-          dpi_dx.get(i) *= -1.0;
-        }
-        return dpi_dx;
-      }(),
-      [&npts, &x, &time, &solution ]() noexcept {
-        tnsr::ij<DataVector, Dim, Frame::Inertial> d2psi_dxdx{pow<Dim>(npts),
-                                                              0.0};
-        const tnsr::ii<DataVector, Dim, Frame::Inertial> ddpsi_soln =
-            solution.d2psi_dxdx(x, time);
-        for (size_t i = 0; i < Dim; ++i) {
-          for (size_t j = 0; j < Dim; ++j) {
-            d2psi_dxdx.get(i, j) = ddpsi_soln.get(i, j);
+  auto local_check_du_dt = [&npts, &time, &solution, &x](
+                               const double gamma2,
+                               const double new_constraint) {
```

Comment:
Can this be renamed "constraint" without the "new_" ? Or is there a particular reason to have the "new" prefix? (I am complaining because "new" is vague and also implies a prior, old constraint...)

Reply:
renamed